src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -156,13 +179,48 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
     window.location.href = venueLink;
   }, [redirectionUrl, venueId]);
 
+  const handleRadioEnable = useCallback(() => setIsRadioPlaying(true), []);
+
+  const [showRadioPopover, setShowRadioPopover] = useState(false);
+
   if (!venueId || !venue) return null;
 
   // TODO: ideally this would find the top most parent of parents and use those details
   const navbarTitle = parentVenue?.name ?? venue.name;
 
   const profileImage = profile?.pictureUrl || DEFAULT_PROFILE_IMAGE;
 
+  const renderSoundCloudIframe = () => {
+    const radioStation = !!hasRadioStations && radioStations![0];
+
+    return (
+      <iframe
+        title="venueRadio"
+        ref={soundcloudIframeRef}
+        id="sound-cloud-player"
```

Comment:
Do we need to use `id` on this? I generally avoid using it unless we have an explicit need for it.